test/tasks/aws_record/test_table_config.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'aws-record'
+
+module ModelTableConfig
```

Comment:
What does this do/how it is called/tested?

Reply:
The migrate rake script looks for all modules of this name in the db/table_config directory and calls this method. In the migrate rake test class, we mock that search to return this file, and when the method is called, the mock is verified to call the migrate method.

Reply:
Ahh that makes sense.  It might be worth adding a comment somewhere about that, I couldn't trace the magic going on.